### PR TITLE
Use a SQL file for DB setup

### PR DESCRIPTION
**Pull request by nymous**
This PR simplifies the Postgres test setup by using only SQL queries, instead of a mix of shell commands and SQL.
This will also be useful for local setup (we can reference it in the docs).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,9 @@ It is recommended to use [rbenv][rbenv] and [nvm][nvm] to install both language
 ## Development
 
 1. Install PostgreSQL
-2. Create a `lea5` user: `sudo --user postgres createuser lea5 --createdb --pwprompt`, and enter `lea5` as a password
-3. Create a development database: `sudo --user postgres createdb lea5_development --owner=lea5`
-4. Create a test database: `sudo --user postgres createdb lea5_test --owner=lea5`
-5. Clone the project
-6. Edit [`config/database.yml`](config/database.yml) if you chose a different password
+2. Initialize the database (users, databases) by running [`init_db.sql`](.github/workflows/init_db.sql): `sudo --user postgres psql --file ./.github/workflows/init_db.sql`
+3. Clone the project
```
Comment (Letiste) [inline README.md:18]:
```suggestion
2. Clone the project
3. Initialize the database (users, databases) by running [`init_db.sql`](.github/workflows/init_db.sql): `sudo --user postgres psql --file ./.github/workflows/init_db.sql`
```

**Review by Letiste — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -14,11 +14,9 @@ It is recommended to use [rbenv][rbenv] and [nvm][nvm] to install both language
 ## Development
 
 1. Install PostgreSQL
-2. Create a `lea5` user: `sudo --user postgres createuser lea5 --createdb --pwprompt`, and enter `lea5` as a password
-3. Create a development database: `sudo --user postgres createdb lea5_development --owner=lea5`
-4. Create a test database: `sudo --user postgres createdb lea5_test --owner=lea5`
-5. Clone the project
-6. Edit [`config/database.yml`](config/database.yml) if you chose a different password
+2. Initialize the database (users, databases) by running [`init_db.sql`](.github/workflows/init_db.sql): `sudo --user postgres psql --file ./.github/workflows/init_db.sql`
```
Comment (Haransis) [inline README.md:17]: Wow impressive

File: README.md (hunk 1 above)
Comment (nymous) [reply README.md:18]: Good catch!

**Review by Haransis — APPROVED** [review]
All good

**Review by Letiste — APPROVED** [review]
LGTM

**Pull request merged**